module-gui/gui/widgets/header/DeleteAction.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include "DeleteAction.hpp"
+#include "Style.hpp"
+
+#include <Text.hpp>
+#include <Image.hpp>
+#include <i18n/i18n.hpp>
+
+namespace gui::header
+{
+    DeleteAction::DeleteAction(Item *parent)
+        : HBox(parent, 0, 0, style::header::navigation_indicator::w, style::header::navigation_indicator::h)
+    {
+        setAlignment(Alignment(Alignment::Horizontal::Right, Alignment::Vertical::Center));
+        setEdges(RectangleEdge::None);
+        setMargins(Margins(style::header::navigation_indicator::right_left_margin,
+                           style::header::navigation_indicator::top_margin,
+                           0,
+                           0));
+
+        auto wasteBin        = new gui::Image("calllog_delete_24px_W_G");
```

Comment:
this file is in generic `module-gui`, so maybe it's worth renaming the asset to be also generic?

Reply:
You mean ust erasing the `calllog_` prefix?

Reply:
exactly ;)

Reply:
Sure

Reply:
Done

Reply:
Done :)